include/zephyr/net/ieee802154_radio.h

```diff
@@ -620,194 +621,208 @@ enum ieee802154_fpb_mode {
-	 *       the nearest one) CSL RX window, to allow the driver to calculate the proper
-	 *       CSL Phase to the nearest CSL window to inject in the CSL IEs for both transmitted
-	 *       data and ACK frames.
-	 *    3. Use @ref IEEE802154_CONFIG_CSL_PERIOD on each value change to update the current
-	 *       CSL period value which will be injected in the CSL IEs together with the CSL Phase
-	 *       based on @ref IEEE802154_CONFIG_CSL_RX_TIME.
-	 *    4. Use @ref IEEE802154_CONFIG_RX_SLOT periodically to schedule the immediate receive
-	 *       window early enough before the expected window start time, taking into account
-	 *       possible clock drifts and scheduling uncertainties.
-	 *
-	 *  This diagram shows the usage of the four options over time:
-	 *        Start CSL                                  Schedule CSL window
-	 *
-	 *    ENH_ACK_HEADER_IE                        CSL_RX_TIME (following window)
-	 *         |                                        |
-	 *         | CSL_RX_TIME (nearest window)           | RX_SLOT (nearest window)
-	 *         |    |                                   |   |
-	 *         |    | CSL_PERIOD                        |   |
-	 *         |    |    |                              |   |
-	 *         v    v    v                              v   v
-	 *    ----------------------------------------------------------[ CSL window ]-----+
-	 *                                            ^                                    |
-	 *                                            |                                    |
-	 *                                            +--------------------- loop ---------+
-	 *
-	 *  @note Available in any interface operational state.
-	 *
-	 *  @note This configuration option does not conform to the requirements
-	 *  specified in #61227 as it is incompatible with standard primitives
-	 *  and may therefore be deprecated in the future.
+	/**
+	 * Configure CSL receiver (Endpoint) period.
+	 *
+	 * @details In order to configure a CSL receiver the upper layer should combine several
+	 * configuration options in the following way:
+	 * 1. Use @ref IEEE802154_CONFIG_ENH_ACK_HEADER_IE once to inform the driver of the
+	 *    short and extended addresses of the peer to which it should inject CSL IEs.
+	 * 2. Use @ref IEEE802154_CONFIG_CSL_RX_TIME periodically, before each use of
+	 *    @ref IEEE802154_CONFIG_CSL_PERIOD setting parameters of the nearest CSL RX window,
+	 *    and before each use of IEEE_CONFIG_RX_SLOT setting parameters of the following (not
+	 *    the nearest one) CSL RX window, to allow the driver to calculate the proper
+	 *    CSL Phase to the nearest CSL window to inject in the CSL IEs for both transmitted
+	 *    data and ACK frames.
+	 * 3. Use @ref IEEE802154_CONFIG_CSL_PERIOD on each value change to update the current
+	 *    CSL period value which will be injected in the CSL IEs together with the CSL Phase
+	 *    based on @ref IEEE802154_CONFIG_CSL_RX_TIME.
```

Comment:
very much a nit but does it need to be spelled "Phase" with a capital P?

Reply:
Not at all. Just an oversight. Fixed.